pinot-transport/src/main/java/com/linkedin/pinot/transport/scattergather/ScatterGatherImpl.java

```diff
@@ -501,14 +501,13 @@ private void runInternal() {
         LOGGER.debug("Response Future is : {}", _responseFuture);
         error = false;
       } catch (TimeoutException e1) {
-        LOGGER.info("Timed out waiting for connection for server ({})({})(gotConnection={}). Setting error future",
-            _server, _request.getRequestId(), gotConnection, e1.getMessage());
+        LOGGER.warn("Timed out waiting for connection for server ({})({})(gotConnection={}). See metric {}", _server,
+            _request.getRequestId(), gotConnection, e1.getMessage(), BrokerMeter.REQUEST_DROPPED_DUE_TO_CONNECTION_ERROR.toString());
```

Comment:
this one has incorrect number of params. e1.getMessage() seems to be missed from log string